### ISSUE TITLE
[Solved] tty0tty error on Linux Mint 19.3

### ISSUE THREAD

**Issue by magellan-13016** [freemed/tty0tty#23]
Hi,
On the last Linux Mint 19.3 with Kernel 5.0.0-37, it is impossible to install tty0tty. 
I get this error : modprobe: ERROR: could not insert 'tty0tty': Device or resource busy

Please help me because without tty0tty, I can't make my educational project with students. I'm a radioamateur and I really need it for Doppler correction with my radio YAESU FT-847.

tty0tty is a part of a very complex software bundle since many years to enable the lack of my radio to be controlled in TX and RX on same frequencies range for ISS communications. I really really need it. 

Have a look at my website : https://f4bpp.com/articles.php?lng=en&pg=55&tconfig=0

Thanks for your help.

David.

**Comment by magellan-13016** [comment]
Problem Solved with installation and activation of new Kernel 5.3.0-24 !

**Comment by stappersg** [comment]
Thanks for report the succes.

**Comment by magellan-13016** [comment]
Problem still not resolved....
I thought it was but in fact not. The problem still remains with new kernel update. I've soved the problem with changing the way to insert the module in the old kernel... sometimes it worked and sometime not so I had to retry until it worked but now, there is no way to load it.

Here is my script :

> cd module
> make
> sudo cp tty0tty.ko /lib/modules/$(uname -r)/kernel/drivers/misc/
> 
> echo Loading module with modprobe
> 
> sudo depmod
> sudo modprobe tty0tty
> sudo /etc/init.d/udev restart
> 
> echo Loading module with insmod
> 
> sudo insmod /lib/modules/$(uname -r)/kernel/drivers/misc/tty0tty.ko

And the result :

insmod: ERROR: could not insert module /lib/modules/5.3.0-26-generic/kernel/drivers/misc/tty0tty.ko: Device or resource busy

Please help !

David.

**Comment by magellan-13016** [comment]
I downloaded the good and last one version of tty0tty following this link : https://github.com/freemed/tty0tty
I think my archive was an old version that was working on old Mint distribution.
With the new one I have downloaded, things worked at the first time.

So the problems was :
1) Kernel issue
2) Old version of tty0tty.

Sorry sorry to have bothered you with this problem.

For people having same issues, I invite you to download the last version here on freemed and update the kernel to the last available.
